### PR TITLE
leave focus of search bar

### PR DESCRIPTION
**Pull request by 3nids**
otherwise, the virtual keyboard is shown when you close the dash board

### CODE REVIEW

File: src/qml/LocatorItem.qml [hunk 1]
```diff
@@ -10,6 +10,11 @@ Item {
   property bool searching: false
   height: childrenRect.height
 
+  function leaveSearching() {
```
Comment (m-kuhn) [inline src/qml/LocatorItem.qml:13]: Can you add a docstring when this should be called?

Comment (3nids) [reply src/qml/LocatorItem.qml:15]: done

Comment (m-kuhn) [reply src/qml/LocatorItem.qml:15]: Just wondering.
Should we use

```
onSearchingChanged {
    if ( !searching )
        focus = false
}
```
 instead? That would effectively avoid misusage

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-a8b099f6da18775e52bae93aeb84e8cd4fd3fd5a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-a8b099f6da18775e52bae93aeb84e8cd4fd3fd5a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-a8b099f6da18775e52bae93aeb84e8cd4fd3fd5a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-a8b099f6da18775e52bae93aeb84e8cd4fd3fd5a-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-4383f993ad3a504e046310b0f4f2b1af36ccd40e-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-4383f993ad3a504e046310b0f4f2b1af36ccd40e-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-4383f993ad3a504e046310b0f4f2b1af36ccd40e-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-4383f993ad3a504e046310b0f4f2b1af36ccd40e-x86_64.apk)

File: src/qml/LocatorItem.qml (hunk 1 above)
Comment (3nids) [reply src/qml/LocatorItem.qml]: done

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-2acfaefec5a7f4cb75b30c5732d5eb305ea35c2a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-2acfaefec5a7f4cb75b30c5732d5eb305ea35c2a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-2acfaefec5a7f4cb75b30c5732d5eb305ea35c2a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-931-2acfaefec5a7f4cb75b30c5732d5eb305ea35c2a-x86_64.apk)

**Pull request merged**